opentasks/src/main/java/org/dmfs/tasks/utils/BaseActivity.java

```diff
@@ -83,24 +81,15 @@ protected void onSaveInstanceState(Bundle outState)
     protected void onPause()
     {
         super.onPause();
-        /*
-         * On older SDK version we have to store permanent data in onPause(), because there is no guarantee that onStop() will be called.
-		 */
-        if (VERSION.SDK_INT < VERSION_CODES.HONEYCOMB)
-        {
-            RetentionMagic.persist(this, mPrefs);
-        }
+        RetentionMagic.persist(this, mPrefs);
```

Comment:
This should be removed. It was only enabled on SDK<11.

Reply:
Good catch, thanks, I've corrected it.

Squashed and rebased, ready for check.